example/InstallExample.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+include __DIR__ . "/../vendor/autoload.php";
+
+use PennyBlack\Api;
+use GuzzleHttp\Psr7\HttpFactory;
+use GuzzleHttp\Client;
+
+$httpClient = new Client();
+$streamFactory = new HttpFactory();
+$requestFactory = new HttpFactory();
+
+$apiKey = "YOUR-API-KEY";
+$isTest = true;
+
+$api = new Api($httpClient, $requestFactory, $streamFactory, $apiKey, $isTest);
+
+$api->install("store.pennyblack.io");
```

Comment:
```suggestion
$api->install("mystore.example.com");
```

for complete clarity that this isn't some fixed Penny Black domain that would be used